### PR TITLE
Warn if environment is misconfigured

### PR DESCRIPTION
**Pull request by jerbri**

### CODE REVIEW

File: CertiPay.Common/EnvUtil.cs [hunk 1]
```diff
@@ -26,15 +28,18 @@ public enum Environment
...
         public static Boolean IsProd { get { return Environment.Production == Current; } }
 
+        private static readonly ILog Log = LogManager.GetLogger(typeof(EnvUtil).Name);
+
         private static readonly Lazy<Environment> current = new Lazy<Environment>(() =>
         {
             Environment environment = Environment.Local;
 
-            String envString = ConfigurationManager.AppSettings["Environment"].TrimToNull() ?? "Local";
+            String envString = ConfigurationManager.AppSettings["Environment"] ?? "Local";
```
Comment (mattgwagner) [inline CertiPay.Common/EnvUtil.cs]: My OCD likes having the `TrimToNull()` here for the cases wherein the config key is there, but no value provided. Not a common scenario, but might as well maintain.

File: CertiPay.Common/EnvUtil.cs [hunk 2]
```diff
@@ -26,15 +28,18 @@ public enum Environment
 
         public static Boolean IsProd { get { return Environment.Production == Current; } }
 
+        private static readonly ILog Log = LogManager.GetLogger(typeof(EnvUtil).Name);
```
Comment (mattgwagner) [inline CertiPay.Common/EnvUtil.cs]: This works, there's also the other overload of `GetLogger<EnvUtil>()`?

**Comment by mattgwagner** [comment]
Good call, kind of obvious in retrospect.

File: CertiPay.Common/EnvUtil.cs (hunk 2 above)
Comment (jerbri) [inline CertiPay.Common/EnvUtil.cs]: Doesn't work on static classes.

Comment (mattgwagner) [inline CertiPay.Common/EnvUtil.cs]: Ah, good to know. Thanks.

**Pull request merged**